README.md

```diff
@@ -1564,6 +1567,110 @@ for domain in domains:
     print(domain)
 ```
 
+### Create Static brcode
+
+Dynamic Brcodes store account information in the form of a Pix Key and can easily be used to create Pix transactions.
```

Comment:
Static BR Codes store account information via a code or an image that represents a Pix Key and a few extra fixed parameters, such as an amount and a reconciliation ID. They can easily be used to receive Pix transactions.